content/specs/machine-type.md

```diff
@@ -0,0 +1,29 @@
+---
+description: How to configure build machine type
+title: Build machine type
+weight: 1
+aliases:
+    - '../building/machine-type/'
+---
+
+## Configuring build machine type
+
+In `codemagic.yaml`, the build machine type can be specified with [Instance type](../getting-started/yaml#instance-type).
+
+For Flutter projects configured via the Flutter workflow editor, the build machine type can be selected in **App settings > Workflow settings > Machine**.
+
+## Mac Mini and Mac Pro
+
+Codemagic offers two types of macOS machines for running builds: Mac Mini (macOS standard VM, default) and Mac Pro (macOS premium VM). Specifications for these machines are available [here](../specs/versions/#hardware). 
```

Comment:
`Mac mini`

Reply:
Why is it `Mac mini` and `Mac Pro`? Should it also be `Mac pro`?

Reply:
trademark I guess: `Mac mini` but `Mac Pro`